hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/safemode/HealthyPipelineSafeModeRule.java

```diff
@@ -118,7 +118,7 @@ protected void process(Pipeline pipeline) {
     Preconditions.checkNotNull(pipeline);
     if (pipeline.getType() == HddsProtos.ReplicationType.RATIS &&
         pipeline.getFactor() == HddsProtos.ReplicationFactor.THREE &&
-        pipeline.isHealthy() &&
+      //  pipeline.isHealthy() &&
```

Comment:
do we want to remove this?

Reply:
I understand now why we can remove this, we trigger open Pipeline event when healthy only. Can we remove the code comment completely.